### ISSUE TITLE
Readme could be amended to add that this utility also fixes the small preview windows opening off top of Win11 screen!

### ISSUE THREAD

**Issue by jonathanmc** [Naamloos/TopCenterStart11#8]
In Windows 11, once you move the taskbar to the top of the screen, Windows 11 opens the small preview windows above that. That is to say, you can't see them - they're above the screen. Thanks, Microsoft.

This utility fixes that!

This should be mentioned in the readme! It's fantastic! I didn't want to tinker but I am happy to fork and amend the readme, if this is seen to be a desired amendment.

**Comment by Naamloos** [comment]
:) Thanks!

I still intend to attempt to make this program independent of the registry tweaks, as Microsoft seems to have remove those by the way.

Feel free to open a PR if I don't make this README change myself in time.

**Comment by jonathanmc** [comment]
I did a quick add just to see how Github handles edits to README. Feel free to edit as you want though, of course!

**Comment by jonathanmc** [comment]
(I see, it forks, makes a branch, and the PR comes from that patch branch. Figured.)

**Comment by jonathanmc** [comment]
I just discovered that this utility also solves the "drag and drop to taskbar" problem!!  Boy you really solved a lot with this utility!! Thanks again!

**Comment by Naamloos** [comment]
> I just discovered that this utility also solves the "drag and drop to taskbar" problem!! Boy you really solved a lot with this utility!! Thanks again!

... it did? xD

**Comment by jonathanmc** [comment]
Ah, I might have spoken too soon. It might be a combination of two utilities that fixes that. Testing more now.

**Comment by jonathanmc** [comment]
Yeah, if you install your fix to fix the "screenshot off top of screen", then this one https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix works to fix drag and drop.

Oh boy, Windows. Getting hacked back to utility, haha.

**Comment by Naamloos** [comment]
aha, yeah I use that tool myself as well.

**Comment by HerMajestyDrMona** [comment]
TopCenterStart11 is a very nice program!

I posted some extra details here: https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/issues/70#issuecomment-1061368908

Tl;dr:

When the taskbar is on top, the Windows11DragAndDropToTaskbarFix works better when the following [configuration line](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/main/CONFIGURATION.md) is added:
```
DetectKnownPixelColorsToPreventAccidentalEvents=0
```

Greetings.

**Comment by Naamloos** [comment]
Thanks for your addition! I'll reopen this issue to add this info later on as well.

**Comment by Naamloos** [comment]
I'll append this to the README on the rewrite branch. A new fully reworked version of the application should be out soon.

**Comment by Naamloos** [comment]
I'm adding this to the README on the rewrite branch. Should be merged soon.
